guides/rhai_expressions_cheat_sheet.cheatmd

```diff
@@ -0,0 +1,54 @@
+# Rhai expressions cheatsheet
+
+We are grouping into this cheatsheet the most frequent manipulations that can be done through Rhai, for convenience.
+
+## Cheatsheet
+{: .col-2}
+
+### Lists
+
+#### Filtering a list
+
+```rust
+[1, 2, 3].filter(|value| value % 2 == 0)
+=> [2]
+```
+
```

Comment:
Can we add links to the Rhai documentation for each of these? Would be handy in case you run into problems. For an experienced programmer it's probably fine to guess the structure/syntax of these, but for someone not that experienced a pointer to the documentation would be great.